api-specs/test/api.raml

```diff
@@ -76,6 +76,46 @@ uses:
           body:
             type: objects.ObjectTestType
             example: !include examples/object-test-type.json
+    head:
+      displayName: Get all resources
+      description: Query/retrieve (HEAD) a specific resource or resource(s). This should return a paged query result of _blah_.
```

Comment:
Nit: I think this should be rephrased. There is no data returned.

Reply:
What would you write instead?

Reply:
@timonrey  maybe an opportunity to learn what the HEAD HTTP request method actually is ;-) 

-> does never return a body, just the headers, hence the name.